### ISSUE TITLE
Report installation time in logs as well

### ISSUE THREAD

**Issue by gadenbuie** [rstudio/renv#1104]
This is an admittedly minor thing, but I've always found it confusing that renv logs report how long it takes to move a package into the cache, but not how long it takes to install the package.

For example, installing `nmle` here took about 10-20 seconds

```
Installing nlme [3.1-160] ...
```

and when the installation is complete it's surprising to see only 0.59ms in the logs.

```
Installing nlme [3.1-160] ...
	OK [built from source]
Moving nlme [3.1-160] into the cache ...
	OK [moved to cache in 0.59 milliseconds]
```

Could you include the installation time for packages, e.g

```
Installing nlme [3.1-160] ...
	OK [built from source in 18 seconds]
Moving nlme [3.1-160] into the cache ...
	OK [moved to cache in 0.59 milliseconds]
```

(It's even worse for a package like `sass` that takes more than a minute to compile from source on my machine but then moves into the cache in half a millisecond.)

**Comment by kevinushey** [comment]
Thanks! Implemented in the development version now:

```
> renv::install("tidyverse", rebuild = TRUE)
The following package(s) are missing their DESCRIPTION files:

	rstan [/Users/kevin/Library/R/arm64/4.2/library/rstan]

These may be left over from a prior, failed installation attempt.
Consider removing or reinstalling these packages.

Retrieving 'https://cran.rstudio.com/bin/macosx/big-sur-arm64/contrib/4.2/tidyverse_1.3.2.tgz' ...
	OK [file is up to date]
Installing tidyverse [1.3.2] ...
	OK [installed binary in 0.42 seconds]
Copying tidyverse [1.3.2] into the cache ...
	OK [copied to cache in 42 milliseconds]
```

**Comment by kevinushey** [comment]
I wonder if we should also include a closing message like "Done! renv::install() finished in 5 seconds." or something as well too -- would that be useful?

**Comment by gadenbuie** [comment]
Awesome, thank you! I do think it'd be helpful to report overall time – I know I'd use it at least!

**Comment by kevinushey** [comment]
Implemented now in renv 0.17.0:

```
> renv::install("rlang")
Retrieving 'https://cloud.r-project.org/bin/windows/contrib/4.3/rlang_1.0.6.zip' ...
	OK [downloaded 1.5 Mb in 0.34 seconds]
Installing rlang [1.0.6] ...
	OK [installed binary in 0.6 seconds]
Moving rlang [1.0.6] into the cache ...
	OK [moved to cache in 7.4 milliseconds]
* Installed 1 package in 0.65 seconds.
```

**Comment by kevinushey** [comment]
Of course, now I see that we're only reporting the time spent installing packages, not retrieving packages, which is misleading -- I just fixed that on main. :doh:
